### ISSUE TITLE
NodeRequire type declaration conflicts.

### ISSUE THREAD

**Issue by sinkersan** [NativeScript/NativeScript#7841]
**Environment**
 - CLI: 6.1.1
 - Cross-platform modules:  6.1.1
 - Android Runtime:  6.1.0
 - iOS Runtime:  6.1.1

**Describe the bug**
NodeRequire interface declaration in tns-core-modules/module.d.ts confilcts with @types/webpack-env
Typescript returns error: 
```
Error:(115, 11) TS2430: Interface 'NodeRequire' incorrectly extends interface 'RequireFunction'.
  Types of property 'context' are incompatible.
    Type '(path: string, deep?: boolean | undefined, filter?: RegExp | undefined) => RequireContext' is not assignable to type '(path: string, deep?: boolean | undefined, filter?: RegExp | undefined, mode?: "sync" | "eager" | "weak" | "lazy" | "lazy-once" | undefined) => RequireContext'.
      Property 'id' is missing in type 'RequireContext' but required in type '__WebpackModuleApi.RequireContext'.
```
**To Reproduce**
Start new Vue project with typescript and install nativescript-vue cli plugin. 
npm run serve:ios (or serve:android)
Error should appear in the build logs.

**Expected behavior**
NodeRequire declaration should not cause any type errors.

**Comment by tsonevn** [comment]
Hi @sinkersan,

I tested your case within a sample project while using the {N} CLI and was unable to recreate such an issue. This problem seems to be more likely related to vue CLI. On that matter, I would suggest opening a new issue in the  https://github.com/nativescript-vue/nativescript-vue/issues

**Comment by sinkersan** [comment]
Hi @tsonevn 
How this can be releated to vue cli or etc if  ts compiler shows error in node-modules/tns-core-modules/module.d.ts file?

**Comment by NathanWalker** [comment]
Reopening as this still exists.

**Comment by NathanWalker** [comment]
Fix Published with @nativescript/core 7.2.1
